src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Menus.vsct

```diff
@@ -97,7 +97,9 @@
     </Groups>
 
     <Buttons>
-      <Button guid="guidManagedProjectSystemOrderCommandSet" id="cmdidMoveUp" priority="0x100">
+      <!-- These buttons are temporarily commented out as we would have duplicate context menu items for move up/down in F# projects. 
```

Comment:
Please remove the code instead of commenting it out. Source control gives history.

Reply:
ok